Turbo/build_lib.sh

```diff
@@ -9,4 +9,4 @@ mkdir -p build
 
 cc -g -fPIC -c -mmacosx-version-min=10.9 -o build/makePathFromOutline.o makePathFromOutline.m
 
-cc -dynamiclib -mmacosx-version-min=10.9 -o ../Lib/fontgoggles/mac/libmakePathFromOutline.dylib -framework AppKit -arch x86_64 -lsystem.b build/makePathFromOutline.o
+cc -dynamiclib -mmacosx-version-min=10.9 -o ../Lib/fontgoggles/mac/libmakePathFromOutline.dylib -framework AppKit -arch x86_64 -arch arm64 -lsystem.b build/makePathFromOutline.o
```

Comment:
This works, but I still get a linker warning if the architectures are only specified in this line. I suggest to add `-arch x86_64 -arch arm64` to the previous line, too.